### PR TITLE
Fixes #1069:Fixed security vulnerability in hapijs/hoek

### LINKED ISSUE #1069: Security Vulnerability in hapijs/hoek
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
Currently there is a security vulnerability in hapijs/hoek dependency defined in `pakage-lock.json`.
**Expected behavior:**
There should be no security vulnerability in project.

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1069 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
I have fixed security vulnerability in hapijs/hoek by updating the dependencies in `package.json` and `package-lock.json`
Surge Link: https://pr-1070-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by praveenojha33** [comment]
@simsausaurabh Please review.

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "codecov.io": "^0.1.6",
     "core-js": "^2.4.1",
     "express": "^4.16.3",
+    "hoek": "^5.0.3",
```
Comment (simsausaurabh) [inline package.json]: Why did you add this dependency additionally ?

File: package-lock.json [hunk 2]
```diff
@@ -9100,6 +9114,13 @@
       "integrity": "sha1-ZUEYTMkK7qbG57NeJlkIJEPGYZg=",
       "requires": {
         "hoek": "2.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```
Comment (simsausaurabh) [inline package-lock.json]: This version is vulnerable.

File: package-lock.json [hunk 3]
```diff
@@ -4684,6 +4691,13 @@
         "cryptiles": "2.x.x",
         "hoek": "2.x.x",
         "sntp": "1.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```
Comment (simsausaurabh) [inline package-lock.json]: This version is vulnerable, and why is it being added here.

File: package-lock.json [hunk 4]
```diff
@@ -1161,6 +1161,13 @@
       "integrity": "sha1-OciRjO/1eZ+D+UkqhI9iWt0Mdm8=",
       "requires": {
         "hoek": "2.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```
Comment (simsausaurabh) [inline package-lock.json]: This version is vulnerable, and why is it being added here?

File: package.json (hunk 1 above)
Comment (praveenojha33) [reply package.json]: I updated the hoek dependency using `npm` and everything was done by dependency manager npm only.

File: package-lock.json (hunk 2 above)
Comment (praveenojha33) [reply package-lock.json]: It is done by `npm` I have checked it by making a commit in my own repository and the message was not coming now.
image

File: package-lock.json (hunk 3 above)
Comment (praveenojha33) [reply package-lock.json]: See above reply.

File: package-lock.json (hunk 4 above)
Comment (praveenojha33) [reply package-lock.json]: See above reply.

File: package-lock.json (hunk 2 above)
Comment (simsausaurabh) [reply package-lock.json]: @praveenojha33 Please see: https://nodesecurity.io/advisories/566. And the message is the alert generated by Github's CVV report. I am not sure why is it not showing alert FYI, as npm suggested. Could you please once check in the dependency settings of the above project, if there seems any issue?

File: package.json (hunk 1 above)
Comment (simsausaurabh) [reply package.json]: What command did you use ?
